platform/os/process-kill-parallel_test.go

```diff
@@ -0,0 +1,88 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	"github.com/litmuschaos/litmus-e2e/pkg"
+	"github.com/litmuschaos/litmus-e2e/pkg/environment"
+	"github.com/litmuschaos/litmus-e2e/pkg/log"
+	"github.com/litmuschaos/litmus-e2e/pkg/types"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/klog"
+)
+
+func TestGoProcessKillParallel(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+//BDD for testing experiment
+var _ = Describe("BDD of process-kill experiment", func() {
+
+	// BDD TEST CASE 1 - process-kill in parallel mode
+	Context("Check for process-kill experiment", func() {
+
+		It("Should check for the process kill in parallel", func() {
```

Comment:
Can we merge the serial/parallel test into one test package `process-kill_test.go` and have them as different test cases as we have followed in other tests.

Reply:
We can't do that as we need different PIDs for both parallel and serial cases, since once a PID is targetted it gets removed and it can't be targetted again